gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,20 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm,' '')"
```

Comment:
You can simplify this a bit:
```suggestion
	local backends="$(usev drm 'drm,')"
```
etc

Reply:
A great idea! Will be done

Reply:
Done in https://github.com/gentoo/gentoo/commit/ad83b87f3f988a2cf276e4a2659c241c48b0bfb1